tests/integration/helpers.py

```diff
@@ -351,6 +359,21 @@ async def check_cluster_formation_successful(
     return set(unit_names) == set(registered_nodes)
 
 
+async def is_up(ops_test: OpsTest, unit_ip: str) -> bool:
+    """Return if node up."""
+    try:
+        for attempt in Retrying(
+            stop=stop_after_attempt(15), wait=wait_fixed(wait=5) + wait_random(0, 5)
```

Comment:
Unless `wait_random(0, 5)` is necessary this is easier to understand and gives reliably timed retries

```suggestion
            stop=stop_after_attempt(15), wait=wait_fixed(wait=5) 
```

Reply:
Though not necessary in this specific test, I would like to stay consistent with the other helpers where a bit of randomness is needed

Reply:
why is the randomness needed elsewhere?

Reply:
I don't remember exactly the [context](https://github.com/canonical/opensearch-operator/pull/12) that compelled me to do so. Regardless, those are tests' helpers - having randomness is not a negative. Would be worth revisiting in the future to trace back the original reasons and see if they're still relevant.